src/status_im/ui/screens/communities/communities_list_redesign.cljs

```diff
@@ -28,14 +28,15 @@
 (def view-type   (reagent/atom  :card-view))
 (def sort-list-by (reagent/atom :name))
 
-(def community-item-data
+(def mock-community-item-data ;; TO Do remove once communities are loaded with this data.
   {:data {:status          :gated
           :locked         true
           :cover          (resources/get-image :community-cover)
           :tokens         [{:id  1 :group [{:id 1 :token-icon (resources/get-image :status-logo)}]}]
           :tags           [{:id 1 :tag-label (i18n/label :t/music) :resource (resources/get-image :music)}
                            {:id 2 :tag-label (i18n/label :t/lifestyle) :resource (resources/get-image :lifestyle)}
                            {:id 3 :tag-label (i18n/label :t/podcasts) :resource (resources/get-image :podcasts)}]}})
+(defn navigate-to [props] #(re-frame/dispatch [:navigate-to :community-overview props]))
```

Comment:
it doesn't make sense to hide even in the function, just use dispatch directly with `>evt`